setup.cfg

```diff
@@ -1 +1,12 @@
-[bdist_wheel]
\ No newline at end of file
+# See the docstring in versioneer.py for instructions. Note that you must
+# re-run 'versioneer.py setup' after changing this section, and commit the
+# resulting files.
+
+[versioneer]
+VCS = git
+style = pep440
+versionfile_source = odm2api/_version.py
+versionfile_build = odm2api/_version.py
+tag_prefix = v
+parentdir_prefix =
+
```

Comment:
Note that most of this PR is boilerplate. What needs reviweing is in this file.
Here we are using the prefix `v` for the version number in the tags, like `v1.0.0`. A tag like `1.0.0` would be ignored by this configuration! And we are also conforming to the [`PEP440`](https://www.python.org/dev/peps/pep-0440/) for version identification and dependency specification. If you agree with those than it should be OK to merge.